Dockerfile

```diff
@@ -1,26 +1,42 @@
-
 # BUILD BIN
 
-FROM golang:1.22.0 as builder
+FROM golang:1.20.0 as app-builder
+
+# Build the app binary in /app
+WORKDIR /app
+
 # Install go modules
-WORKDIR /flow-evm-gateway
 COPY go.* ./
 COPY . ./
+
 RUN go mod download
 RUN go mod verify
-RUN CGO_ENABLED=1 go build -o evm-gateway ./cmd/main/main.go
-RUN chmod a+x evm-gateway
-RUN chmod a+x ./scripts/run.sh
+
+# Build binary
+RUN CGO_ENABLED=1 go build -o bin ./cmd/main/main.go
+RUN chmod a+x bin
 
 # RUN APP
+
 FROM debian:latest
+
+ENV ACCESS_NODE_GRPC_HOST=""
```

Comment:
Not sure what we can use here as default values, if there are sensible defaults.